pkg/node/refresh.go

```diff
@@ -0,0 +1,18 @@
+package node
+
+import (
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	networkStateRefresh = time.Minute
+)
+
+// NodeNetworkStateRefresh add some jitter to to the refresh rate so it does
+// not hit apiserver at the same time. it returns random time.Duration in range
+// [60-66] seconds.
```

Comment:
The refresh interval is taken from a constant. It should not be documented in the function. It would get out of sync and frankly, the default value should be irrelevant while you are coding. Things should work no matter if it is a second on an hour.